CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Copy and pasting the git commit messages is __NOT__ enough.
 - Interest vehicles now show up in Envision.
 - Seed of `hiway-v1` env can be retrieved through a new property `seed`.
 - Added `TrafficEngineActor` to describe a scenario studio defined actor that is controlled by a traffic engine.
+- Added CI test for RL Drive and VehicleFollowing examples, which tests the RL training process.
```

Comment:
This is not really user-facing, I would consider removing it or adding a dev section. The test can be checked through `git blame`.

Reply:
Reverted the addition in changelog.